config-template.json

```diff
@@ -0,0 +1,10 @@
+{
+	"hostName": "localhost",
+	"port": 8080,
```

Comment:
Does it really work with localhost and port 8080? I always needed to specify  xxx.xxx.xxx.xxx:8080 to get that work. I do not think I ever got localhost:8080 to work either. And I suggest the file should be named config.json for simple upstart.